scss/_equation_editor.scss

```diff
@@ -312,27 +312,30 @@ $menu-handle-width: 160px;
         position: absolute;
         bottom: 18px;
         cursor: pointer;
-        &.centre,
+        // &.centre,
         &.redo,
         &.undo,
+        &.poly,
+        &.straight,
+        &.mirro,
         &.clearAll {
             width: 68px;
             height: 80px;
         }
-        &.centre {
```

Comment:
Does this break the actual equation editor?

Reply:
I've undone these omissions on the merge conflict fix. It doesn't affect the graph sketcher at the moment as we don't use the "centre" button.